src/components/OptionList/components/Option/Option.scss

```diff
@@ -48,6 +48,7 @@ $control-vertical-adjustment: rem(2px);
   cursor: pointer;
   border-radius: var(--p-border-radius-base);
   padding: spacing(tight);
+  word-break: break-word;
```

Comment:
👋 Just a heads up this property is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#syntax)

Better to use `overflow-wrap: anywhere` which should work in the same way

Reply:
As discussed on slack, `overflow-wrap: anywhere` is not an option as it isn't supported by Safari.  Instead, I went with `word-break: break-all` which achieves intended fix and also works with Chinese/Japanese/Korean (CJK) text.